examples/express/app.js

```diff
@@ -0,0 +1,35 @@
+const bugsnag = require('@bugsnag/js')
+const express = require('express')
+const { readFileSync } = require('fs')
+
+const bugsnagClient = bugsnag(process.env.BUGSNAG_API_KEY)
+bugsnagClient.use(require('@bugsnag/plugin-express'))
+
+const app = express()
+const { requestHandler, errorHandler } = bugsnagClient.getPlugin('express')
+
+app.use(requestHandler)
+
+app.use('/static', express.static(`${__dirname}/static`))
+
+const index = readFileSync('./views/index.html', 'utf8')
+app.get('/', (req, res) => {
+  res.send(index)
+})
+
+app.post('/crash', (req, res) => {
+  setTimeout(() => { throw new Error('Uh oh') })
+})
+
+app.post('/handled', (req, res) => {
+  throw new Error('Just a wobble')
```

Comment:
I just noticed that this isn't actually "handled" by Bugsnag's definition. The server stays "up" and the process does not crash, but the error is not handled by the user. We actually want two routes here:

```suggestion
app.post('/unhandled', (req, res) => {
  throw new Error('Just a wobble')
})

app.post('/handled', (req, res) => {
  req.bugsnag.notify(new Error('Could not connect to xyz service'))
  res.sendStatus(503)
})
```